basicLibrary/src/main/java/basicLibrary/lab3.java

```diff
@@ -0,0 +1,64 @@
+package basicLibrary;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class lab3 {
+    public String analyzeWeather(int[][] arr){
+        int count = 1;
```

Comment:
Perhaps I missed it, but I don't think you're using count for anything. You increment it on line 23 but then it's just chilling